cli/azd/cmd/root.go

```diff
@@ -145,6 +145,14 @@ func NewRootCmd(ctx context.Context, staticHelp bool, middlewareChain []*actions
 		},
 	})
 
+	root.Add("vs-server", &actions.ActionDescriptorOptions{
+		Command:        newVsServerCmd(),
+		FlagsResolver:  newVsServerFlags,
+		ActionResolver: newVsServerAction,
+		OutputFormats:  []output.Format{output.NoneFormat},
```

Comment:
I believe you can ommit the output formats and the default format if None is your default.  But also no issues if you just want to be more verbose.